### DISCUSSION TITLE
Date / Year always shown in the vertical scrollbar

### DISCUSSION THREAD

**Discussion by Phisoloc** [ente-io/ente#2236] [Enhancements]
There are several other apps where I can navigate between different dates and years.

For example Samsung Gallery has such a feature in the scroll bar and Synology Photos as well. It would make the ease of navigation much more convenient.

Examples are attatched below:

![Point Blur_Jun212024_070230](https://github.com/ente-io/ente/assets/22276631/7a5d1383-5ebf-472f-b5a0-cbf9c5e299e9)

![Point Blur_Jun212024_070424](https://github.com/ente-io/ente/assets/22276631/b77ee339-8e4e-41f9-a81b-be2e2b1d7b8f)

**Comment by vovchykbratyk** [comment]
Yes, definitely request this across all platforms.

**Comment by wrobelda** [comment]
Immich has that, too, and is web-based as well:
![](https://o-toole.com/wp-content/uploads/2024/11/immich-frontend-scaled.jpg)

**Comment by eagle-zero** [comment]
At least the iOS app has this when you hold and move the slider from the scroll bar. 
But it is sometimes a bit buggy and doesn't show up. 
![IMG_6437](https://github.com/user-attachments/assets/aceb44a9-1d95-477d-b08a-9e1539b296ac)

**Comment by Chistex** [comment]
Would be great to geht this feature.

Also ist would be great to filter our month/year.
